linux_os/guide/services/ntp/service_chronyd_or_ntpd_enabled/rule.yml

```diff
@@ -35,8 +35,6 @@ rationale: |-
 
 severity: medium
 
-platform: machine  # The check uses service_... extended definition, which doesnt support offline mode
```

Comment:
Containers have time services too. Reverting this change.

Reply:
ditto (see `chronyd_or_chronyc_network`)